web/packages/teleterm/src/ui/ConnectMyComputer/NavigationMenu.tsx

```diff
@@ -21,39 +21,62 @@ import { Box } from 'design';
 
 import { useAppContext } from 'teleterm/ui/appContextProvider';
 import { ListItem } from 'teleterm/ui/components/ListItem';
+import { ClusterUri } from 'teleterm/ui/uri';
 
-import { ConnectMyComputerIcon } from './ConnectMyComputerIcon';
+import { NavigationMenuIcon } from './NavigationMenuIcon';
+import { isConnectMyComputerPermittedForRootCluster } from './permissions';
 
-export function ConnectMyComputer() {
+interface NavigationMenuProps {
+  clusterUri: ClusterUri;
+}
+
+export function NavigationMenu(props: NavigationMenuProps) {
```

Comment:
Does it open the popover from within `DocumentCluster`? I'd confirm that with Kenny, I assumed that the new version was supposed to open the status bar instead. But I don't think we talked about this explicitly.

Reply:
Yes, it does. I will ask Kenny if we want to get rid of this.